File: Java/Logica-Programacao/src/faccat/Ex040.java
package faccat;

import java.util.Scanner;

public class Ex040 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Digite o nome do produto: ");
        String name = sc.nextLine();
        System.out.print("Digite a quantidade adquirida: ");
        short qty = sc.nextShort();
        System.out.print("Digite o preço unitário: ");
        float price = sc.nextFloat();
        sc.close();

        float total = qty * price;

        if (qty <= 5) {
            total = total - total * 2 / 100;
        } else if (qty <= 10) {
            total = total - total * 3 / 100;
        } else total = total - total * 5 / 100;

        System.out.printf("O total a pagar da compra do produto %s é R$%.2f.\n", name, total);
    }
}

File: Logica-Programacao/Javascript/Faccat/ex040.js
let nome, quantidade, precoUnidade, desconto, totalPagar;

nome = prompt("Descrição do produto: ")
quantidade = parseInt(prompt("Quantidade adquirida: "))
precoUnidade = parseFloat(prompt("Preço unitário: "))

totalPagar = quantidade * precoUnidade

if (quantidade <= 5) {
    desconto = 0.02
} else if ((quantidade > 5) && (quantidade <= 10)) {
    desconto = 0.03
} else {
    desconto = 0.05
}

totalPagar = totalPagar - (totalPagar * desconto)

alert("Produto: " + nome + " | Quantidade: " + quantidade + " | Desconto: " + desconto + "% | Total a pagar: R$" + totalPagar)